.travis.yml

```diff
@@ -175,9 +175,11 @@ matrix:
       before_install:
         - docker run -d --name clickhouse-server -p 9000:9000 -p 9009:9009 -p 8123:8123 --ulimit nofile=262144:262144 yandex/clickhouse-server
         - docker run -d --env SNUBA_SETTINGS=test --env CLICKHOUSE_SERVER=clickhouse-server:9000 --name snuba -p 1218:1218 --link clickhouse-server:clickhouse-server getsentry/snuba
+        - docker run -d --name zookeeper -e ZOOKEEPER_CLIENT_PORT=2181 confluentinc/cp-zookeeper:4.1.0
+        - docker run -d --name kafka --link zookeeper -p 9092:9092 -e KAFKA_ZOOKEEPER_CONNECT=zookeeper:2181 -e KAFKA_ADVERTISED_LISTENERS=PLAINTEXT://localhost:9092 -e KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR=1 confluentinc/cp-kafka:4.1.0
         - docker ps -a
       install:
-        - pip install -e ".[dev,tests,optional]"
+        - pip install -e ".[dev,tests,optional]" confluent-kafka
```

Comment:
This could probably be an optional dependency (and should be pinned, more than likely) but I don't really want to impose this on everyone locally. Maybe a separate group for Kafka?